src/cdk/explorer/tree/treeInspector.ts

```diff
@@ -63,3 +63,15 @@ export function getDisplayLabel(construct: ConstructTreeEntity): string {
 
     return construct.id
 }
+
+
+/** 
+* (for CDK Visualization) Determines if a construct is a Resource and a state machine
```

Comment:
We generally don't want to specify that this function is for CDK visualization. The function may be used for multiple different features.